pkg/schedule/checker/rule_checker.go

```diff
@@ -132,6 +132,11 @@ func (c *RuleChecker) CheckWithFit(region *core.RegionInfo, fit *placement.Regio
 		return
 	}
 
+	// the placement rule is disabled
+	if fit == nil {
+		return
```

Comment:
Do we need to add one metrics for it or add log?

Reply:
I think no need since it happens only once.